### PR TITLE
Properly disable dragging on and left of a locked column

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: src/app/public/modules/grid/grid-adapter.service.ts [hunk 1]
```diff
@@ -49,19 +49,54 @@ export class SkyGridAdapterService {
...
+
+        if (sibling === undefined || !sibling) {
+          return true;
         }
+
+        const columns = source.querySelectorAll('th div');
+        const siblingDiv = sibling.querySelector('div');
+        let isLeftOfLocked = false;
+
+        for (let i = (columns.length - 1); i >= 0; i--) {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/grid/grid-adapter.service.ts]: Any chance we could encapsulate this logic so its not repeated? Maybe something named `isLeftOfLocked()` that returns a boolean?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/grid/grid-adapter.service.ts]: @Blackbaud-AlexKingman done.

File: src/app/public/modules/grid/grid-adapter.service.ts [hunk 2]
```diff
@@ -49,23 +49,50 @@ export class SkyGridAdapterService {
...
+          && !isLeftOfLocked;
+      }
     });
   }
 
   public setStyle(el: ElementRef, style: string, value: string): void {
     this.renderer.setStyle(el.nativeElement, style, value);
   }
+
+  private isLeftOfLocked(sourceColumn: HTMLElement, columns: NodeListOf<Element>) {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/grid/grid-adapter.service.ts]: Needs return type :)

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/grid/grid-adapter.service.ts]: @Blackbaud-AlexKingman done

**Pull request merged**